cssFiles/grid.css

```diff
@@ -0,0 +1,34 @@
+.grid-heading {
+    margin-left: 12rem;
+    font-size: 2rem;
+    font-weight: bold;
+    color: var(--primary-color);
+}
+
+.grid-subheading{
+    margin-left: 11.5rem;
+    margin-top: .25rem;
+}
+
+
+.two-items {
+    grid-template-columns: 1fr 1fr;
+    max-width: 30rem;
+    display: grid;
+    grid-gap: 1rem;
+    margin: auto;
+  }
+  
+  .three-items {
+    grid-template-columns: 1fr 1fr 1fr;
+    max-width: 36rem;
+    display: grid;
+    grid-gap: 1rem;
+    margin: auto;
+  }
+  
+  .grid-items {
+    padding: 1rem;
+    text-align: center;
+    border: 1px solid black;
```

Comment:
can you use this color fro  your root  instead of directly defining them?

Reply:
sure, 
thanks